examples/API_v2/get_users_mentions.py

```diff
@@ -0,0 +1,19 @@
+import tweepy
+
+
+# Replace bearer token value with your own
+bearer_token = ""
+
+# Initializing the Tweepy client
+client = tweepy.Client(bearer_token)
+
+# Replace user ID
+id = '2244994945'
+
+# By default the Tweet ID and Tweet text will be returned. If you want additional data,
+# specify it used fields and expansions
+tweets = client.get_users_mentions(id=id)
+
+for tweet in tweets.data:
+    print(tweet)
```

Comment:
This will print the Tweet cast to a string, which will just be the text of the Tweet.
It's not really clear that that's what this does, without any explanation, and the output ends up being a concatenation of Tweet text that probably isn't very useful.